v21.2/sql-feature-support.md

```diff
@@ -168,16 +168,19 @@ table tr td:nth-child(2) {
  Roles | ✓ | Standard | [Roles documentation](authorization.html#roles)
  Object ownership | ✓ | Common Extension | [Ownership documentation](authorization.html#object-ownership)
  Privileges | ✓ | Standard | [Privileges documentation](authorization.html#assign-privileges)
+ Default privileges | ✓ | PostgreSQL Extension | [Default privileges documentation](authorization.html#default-privileges)
```

Comment:
I would leave this as partial support for now

Reply:
Done